bptestrunner/bluepill_batch_test_runner.template.sh

```diff
@@ -12,14 +12,12 @@ basename_without_extension() {
 # Constants
 TEST_BUNDLE_PATHS=(test_bundle_paths)
 TEST_HOST_PATHS=(test_host_paths)
-BP_WORKING_FOLDER="bp_exec_root"
+BP_WORKING_FOLDER="$TEST_TMPDIR/bp_exec_root"
```

Comment:
I'm not confident that this is necessary (things seem to be working as-is), but https://docs.bazel.build/versions/main/test-encyclopedia.html#test-interaction-with-the-filesystem suggests that test runners should only write to `$TEST_TMPDIR` and `$TEST_UNDECLARED_OUTPUTS_DIR`.